vhost-device-console/README.md

```diff
@@ -119,9 +134,22 @@ host# qemu-system
     ...
 ```
 
+3) Using User Mode Linux [`virtio console`](https://github.com/torvalds/linux/blob/848e076317446f9c663771ddec142d7c2eb4cb43/include/uapi/linux/virtio_ids.h#L34):
+```text
+host# linux root=/dev/ubda1 rw ubd0=$YOUR-PATH/kata-ubuntu-latest.image   \
+    <normal UML options>                                             \
+    virtio_uml.device=/tmp/console.sock0:3 console=tty0 console=hvc0 \
+    init=/bin/systemd                                                \
+    systemd.unit=kata-containers.target agent.debug_console
+```
+Test with [kata-ubuntu-latest.image](https://github.com/kata-containers/kata-containers/releases/),
+you can also use systemd to setup pty manually without kata-agent help.
+
 Eventually, the user can connect to the console by running:
 ```test
-host# stty -icanon -echo && nc localhost 12345 && stty echo
+host# stty -icanon -echo && nc localhost 12345 && stty echo # for network backend
+or
```

Comment:
put the or outside the block or create 2 points

Reply:
ok